docs/docs-components/CombinationNew.tsx

```diff
@@ -86,7 +86,12 @@ export default function CombinationNew({
       cardShadeColor = 'darkWash';
     }
     if (
-      combinationTitles.some((title) => title.includes('"washLight"') || title.includes('"dark"'))
+      combinationTitles.some(
+        (title) =>
+          title.includes('"washLight"') ||
+          title.includes('"dark"') ||
+          title.includes('"transparentDarkGray"'),
```

Comment:
why did we remove this one?

Reply:
No removal, but I see now this change was unnecessary, the old background for transparentDarkGray was correct. I have reverted it.